api/TextMetrics.json

```diff
@@ -602,10 +602,10 @@
               "version_added": false
             },
             "opera": {
-              "version_added": null
+              "version_added": "73"
```

Comment:
The Chrome data suggests this is behind a flag and thus not shipped. Can you update that too if it's not correct?

Reply:
#7989 adds the Chrome data, which adds versions for when it was no longer behind a flag.  Realize I should have put these changes with that PR, sorry!

Reply:
Aha, so Chrome 87, which is the same as Opera 73. Thanks!